pages/tokenDetails.tsx

```diff
@@ -1,15 +1,18 @@
+import { useState } from 'react'
 import PrimaryButton from '@/components/buttons/PrimaryButton'
 import SecondaryButton from '@/components/buttons/SecondaryButton'
 import Container from '@/components/Container'
 import Input from '@/components/Input'
-import Steps from '@/components/steps/Steps'
+import { Steps } from '@/components/steps'
```

Comment:
We just call the steps component entrypoint and don't care about what files are inside the component. We only care about what the entrypoint exports for other components to use.

In this case, we just use the `Steps` function.